bundle/manifests/authorino-operator-metrics_v1_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: authorino-operator
+  name: authorino-operator-metrics
```

Comment:
Im thinking we make the naming the same as it it for  limitador, kuadrant and cert manager to. `<OPERATOR_NAME>-controller-manager-metrics-service` wdyt?

Reply:
I agree with the premise, though IMHO I believe it's Limitador Operator the one that should change.

`controller-manager` is very poor choice for something that is known by the user as "\<X\> Operator", while the `-service` suffix is not needed to identity an object whose kind is already a Service.

Reply:
im neither here nor there about it, i think should all just be the same. I can change the other 3 services to follow suit of this naming if you prefer

Reply:
Operator SDK's the one blame, I think. But yeah. I do prefer plain simple `authorino-operator-metrics` here. In fact, we went that road 2 years ago: #52.